orders/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+class Cart(models.Model):
+    account  = models.ForeignKey('accounts.Account', on_delete=models.CASCADE)
+    option   = models.ForeignKey('products.Option', on_delete=models.CASCADE)
+    quantity = models.IntegerField()
+    added_at = models.DateTimeField(auto_now_add=True)
```

Comment:
```suggestion
    created_at = models.DateTimeField(auto_now_add=True)
```
장바구니에 추가 되었다는 것은 장바구니 데이터가 생성된 것이므로 created_at이 적절할 것 같습니다.
(참고: created_at과 updated_at은 개발자들이 사용하는 일종의 컨벤션입니다.)

Reply:
created_at으로 수정하였습니다.